frontend/src/app/shared/components/datepicker/datepicker.modal.sass

```diff
@@ -3,9 +3,9 @@
 .op-datepicker-modal
   z-index: 500
   width: auto
-  // prevent additional content like notifications to
-  // drastically increase the width.
-  max-width: 730px
+  // Basically the width of the two calendars next to each other + spacings
+  // will be overwritten on mobile
+  max-width: 600px
```

Comment:
Wasn't this `650px`?

Reply:
Yes, but I did some other changes, that reduced that further